### PR TITLE
Update null-object-pattern.md

### PR DESCRIPTION
**Pull request by zzfima**
Add NullObjectCustomer
Remove some typo "c =&gt;"

### CODE REVIEW

File: src/docs/design-patterns/null-object-pattern.md [hunk 1]
```diff
@@ -40,23 +40,41 @@ public class Customer
 }
 ```
 
+Another way implement the Null Object Pattern, is by using inhetance:
```
Comment (ardalis) [inline src/docs/design-patterns/null-object-pattern.md:43]:
```suggestion
Another way implement the Null Object Pattern, is by using inheritance:
```

File: src/docs/design-patterns/null-object-pattern.md [hunk 2]
```diff
@@ -40,23 +40,41 @@ public class Customer
...
+```
+
 Then, wherever you would have a method that could return a null Customer, have it return the static instance instead:
 
 ```java
 public Customer GetByPhoneNumber(string phoneNumber)
 {
  var customer = _customerRepository
-                .List(c =&gt; c.PhoneNumber == phoneNumber)
+                .List(c.PhoneNumber == phoneNumber)
```
Comment (ardalis) [inline src/docs/design-patterns/null-object-pattern.md:62]:
```suggestion
                .List(c => c.PhoneNumber == phoneNumber)
```
The previous syntax was correct. It's a lambda expression in C#.

File: src/docs/design-patterns/null-object-pattern.md [hunk 3]
```diff
@@ -40,23 +40,41 @@ public class Customer
...
                 .FirstOrDefault();
   if(customer == null) { return Customer.NotFound; }
   return customer;
 }
 ```
 
+If using NullObjectCustomer, code is very same, except of:
```
Comment (ardalis) [inline src/docs/design-patterns/null-object-pattern.md:69]:
```suggestion
If using `NullObjectCustomer`, code is the same, except for:
```

**Review by ardalis — CHANGES_REQUESTED** [review]
minor changes please

**Comment by ardalis** [comment]
You don't want to add the java class you showed?

**Comment by zzfima** [comment]
@ardalis i closed it y mistake

**Comment by zzfima** [comment]
i have accept 1 suggestion

**Comment by zzfima** [comment]
@ardalis hi I new in GitHub open source. Shall I do something?

**Comment by ardalis** [comment]
Sorry; got busy. :) 

Thanks!

**Pull request merged**